docs/recipes/Deploying-Modules.md

```diff
@@ -0,0 +1,159 @@
+[👈 Return to Overview](./Recipes.md)
+
+# Deploying Modules
+
+## 📖 Table of Contents
+
+* [Deploying to Now with GitHub Actions](#deploying-to-now-with-github-actions)
+  * [Creating your Deploy Action](#creating-your-deploy-action)
+  * [Create and Update Your Module Map](#create-and-update-your-module-map)
+    * [Creating a Module Map in Now.sh](#creating-a-module-map-in-nowsh)
+    * [Create an Update Module Script](#create-an-update-module-script)
+    * [Update your Deploy Action to Update your Module Map](#update-your-deploy-action-to-update-your-module-map)
+
+* [Deploying to Now with GitHub Actions](#deploying-to-now-with-github-actions)
```

Comment:
Is this just a dupe of line 7?

Reply:
Thanks for catching that... My TOC creator in VSCode must of not updated correctly and I missed that.